msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -380,6 +383,26 @@ protected PublicClientApplication(@NonNull final Context context,
         }
     }
 
+    private void setupTelemetry(@NonNull final Context context,
+                                @Nullable final PublicClientApplicationConfiguration developerConfig) {
+        if (null != developerConfig.getTelemetryConfiguration()) {
```

Comment:
`developerConfig` is marked `Nullable` , needs null check for it as well.

Reply:
Changed form @Nullable to @NonNull. Moved to into setupConfiguration and using mPublicClientConfiguration which is always nonnull.